Menu_Lab/script.js

```diff
@@ -0,0 +1,24 @@
+
+const menuButton = document.getElementById("menu-btn")
+const menuBox = document.getElementsByClassName("menu-box")[0]
+menuButton.addEventListener('click', () => {
+    menuBox.classList.toggle('visible')
+})
+
+const letterLink = document.getElementsByClassName("letter-link")[0]
+const letters = document.getElementsByClassName("letters")[0]
+letterLink.addEventListener('click', () => {
+    letters.classList.toggle('visible')
+    menuBox.classList.toggle('visible')
+    numbers.style.display = 'none'
+    letters.style.display = 'flex'
```

Comment:
```suggestion
    menuBox.classList.toggle('visible')
    numbers.style.display = 'none'
    letters.style.display = 'flex'
```
Toggling the visible class on the letters/numbers isn't needed since you're explicitly doing it with lines 13-14.